docs/content/en/docs/releases/1.13.0.md

```diff
@@ -21,11 +21,11 @@ ImagePullPolicy and PodSpec are parameters which can be used to control the Kube
 #### RequestsPerPod and OnceOnly fields are added for pool manager
 
 RequestsPerPod will denote how many requests will be routed to each pod in Pool Manager.
-OnceOnly is similar to a Kubernetes Job, it is useful in long-running tasks where only one request is served by one pod.
+OnceOnly is similar to a Kubernetes Job, it is useful in long-running tasks where only one request is served by one pod. You can read more about it [here](https://docs.fission.io/docs/usage/executor/)
 
 #### Websocket support for reaping
 
-With recent changes in the NodeJS environment for websocket, conventional timeout based reaping wasn't applicable so an event-based reaping mechanism to support Websocket was introduced.
+With recent changes in the NodeJS environment for websocket, conventional timeout based reaping wasn't applicable so an event-based reaping mechanism to support Websocket was introduced. You can try out the example [here](https://github.com/fission/examples/tree/master/samples/websocket) 
```

Comment:
TODO:
Update the README in examples repo to the blog when it's merged:
https://github.com/fission/blog.fission.io/pull/69

Reply:
cc @gauravgahlot